frontend/front/docker-builds/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# grab nginx image running on alpine linux
+FROM nginx:stable-alpine
+
+# update and add dependencies
+RUN apk update && apk upgrade && apk add openssl curl ca-certificates
+
+# copy contents of build folder to default folder in container
+COPY . /usr/share/nginx/html
```

Comment:
Why not just do `COPY ../build /usr/share/nginx/html` ? Having to copy this into the build folder seems annoying.